### PR TITLE
add factory method to make a tessel command

### PR DESCRIPTION
**Pull request by wprater**
- adds default options for name, usb, lan, and timeout

### CODE REVIEW

**Comment by wprater** [comment]
@johnnyman727 some methods did not have all the `name`, `usb`, `lan`, and `timeout` methods.  for example `erase`.  with the new `makeCommand` method, they will.  is this what you intended?

File: lib/tessel/deploy.js [hunk 1]
```diff
@@ -29,18 +29,7 @@ Tessel.prototype.deployScript = function(opts) {
 
     // Stop running any existing scripts
     return actions.execRemoteCommand(self, 'stopRunningScript', filepath)
-      .then(function() {
```
Comment (johnnyman727) [inline lib/tessel/deploy.js]: @wprater this seems unrelated to this PR. Does it need to be in here?

File: bin/tessel-2.js [hunk 2]
```diff
@@ -38,6 +38,20 @@ function closeFailedCommand(err) {
   process.exit(1);
 }
 
+function makeCommand(commandName) {
+  return parser.command(commandName)
+    .option('timeout', timeoutOption)
+    .option('name', nameOption)
+    .option('lan', {
```
Comment (johnnyman727) [inline bin/tessel-2.js]: I think we should either break the `lan` and `usb` option into separate variables just like `timeoutOption` and `nameOption` are or bring the `timeoutOption` and `nameOption` back into this function (for consistency). I don't think those two vars are used anywhere else.

**Comment by johnnyman727** [comment]
@wprater just two comments but this is a great refactor.

File: lib/tessel/deploy.js (hunk 1 above)
Comment (wprater) [inline lib/tessel/deploy.js]: whoops.. it was from another PR.. will rebase out.

**Comment by wprater** [comment]
good to merge when you are, @johnnyman727

**Comment by johnnyman727** [comment]
Love it!

**Comment by wprater** [comment]
need a quick hotfix!

**Comment by wprater** [comment]
nevermind! I tricked myself while I was on another branch (:
